tests/pii_replacer/test_freetext.py

```
@@ -0,0 +1,237 @@
+                "patient_name": f"{last.upper()}, {first}",
+                "notes": f"Patient {last.upper()} was seen in clinic today for a routine follow up visit {i}",
+            }
+        )
+    df = pd.DataFrame(rows)
+    cfg = ReplacePiiConfig(person=PiiPersonConfig(backend=PiiPersonBackend.faker))
+    replacer = TabularPiiReplacer(cfg, data_config=DataParameters())
+    replacer.transform_df(df)
+    assert replacer.result is not None
+    out = replacer.result.transformed_df
+
+    for new_name, new_note, original in zip(out["patient_name"], out["notes"], df["patient_name"]):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)